test/config.sh

```diff
@@ -124,12 +124,12 @@ assert_dns_record() {
     exp_ips_regex=$(echo "$@" | sed -r 's/ /\\\|/g')
 
     [ -z "$DEBUG" ] || greyly echo "Checking whether $name exists at $host:$container"
-    got_ip=$(exec_on $host $container getent hosts $name)
+    got_ip=$(exec_on $host $container getent hosts $name | awk '{print $1}')
     assert_raises "echo '$got_ip' | grep -q '$exp_ips_regex'"
 
     [ -z "$DEBUG" ] || greyly echo "Checking whether the IPs '$@' exists at $host:$container"
     for ip in "$@" ; do
-        assert "exec_on $host $container getent hosts $ip | tr -s ' '" "$ip $name"
+        assert "exec_on $host $container getent hosts $ip | awk '{print \$2}'" "$name"
```

Comment:
getent hosts was returning the hostname twice, so the output wouldn't match. Could have changed it to be a grep instead, like above.
```
$ getent hosts $ip
10.2.0.78      c1.weave.local      c1.weave.local
```